samples/client/petstore/csharp-netcore/OpenAPIClient-ConditionalSerialization/docs/ModelClient.md

```diff
@@ -4,7 +4,7 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**_Client** | **string** |  | [optional] 
+**Property_Client** | **string** |  | [optional] 
```

Comment:
should this be `PropertyClient` instead of `Property_Client`?

Reply:
No, because the template hardcodes an underscore there. Note that I did not change templates for RestSharp.